### PR TITLE
Added list oders

### PR DESCRIPTION
**Pull request by even1111112**

### CODE REVIEW

File: tests/test_routes.py [hunk 1]
```diff
@@ -70,6 +70,15 @@ def test_index(self):
     ######################################################################
     #  O R D E R S  T E S T  C A S E
     ######################################################################
+
+    def test_list_orders(self):
```
Comment (nicksome168) [inline tests/test_routes.py:74]: add doc string

Comment (even1111112) [reply tests/test_routes.py:74]: > add doc string

Done.

Comment (nicksome168) [reply tests/test_routes.py:74]: what i meant was this:

```
def test_list_orders(self):
        """It should List all Orders"""
        ...
```

This doc string will show up when we run the green test

Comment (even1111112) [reply tests/test_routes.py:74]: > what i meant was this:
> 
> ```
> def test_list_orders(self):
>         """It should List all Orders"""
>         ...
> ```
> 
> This doc string will show up when we run the green test

Have modified

**Pull request merged**